src/eggd_sompy.sh

```
@@ -31,13 +31,36 @@ main() {
+            normalised_query_vcf="${query_vcf_name}.normalized.vcf.gz"
+            bcftools norm \
+                -Oz \
+                -W=tbi \
+                -f ${reference_file} \
+                -m -any \
+                --keep-sum AD \
+                -o "${normalised_query_vcf}" \
+                "${query_vcf}"
+
+            # normalise truth VCF
+            truth_vcf_name=$(sed -E 's/\.vcf(\.gz)?//g' <<< $(basename $truth_vcf))
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)